public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -242,7 +242,9 @@ function createRowItemFromLegacyRow(row: PanelModel, panels: DashboardGridItem[]
     layout: new DefaultGridLayoutManager({
       grid: new SceneGridLayout({
         // If the row is collapsed it will have panels within the row model.
-        children: (row.panels?.map((p) => buildGridItemForPanel(p)) ?? []).concat(panels),
+        children: (
+          row.panels?.map((p) => buildGridItemForPanel(p instanceof PanelModel ? p : new PanelModel(p))) ?? []
```

Comment:
buildGridItemForPanel must receive PanelModel; this was a bug that was already there, in V1 dashboard